### ISSUE TITLE
bepo layout and delay

### ISSUE THREAD

**Issue by nicodebo** [vifm/vifm#248]
I'm using a bépo layout which is kind of a french version of dvorak, see [bépo](https://download.tuxfamily.org/dvorak/wiki/images/Carte-bepo-simple.png)

Thus, for basic movements I am remapping {h,j,k,l} to {c,t,s,r}.
My vifmrc looks like the following:
```
noremap c h
noremap t j
noremap s k
noremap r l
```

Then when navigating one step higher in the directory tree using `c` key, I have a small delay (~1 second) before the view actually goes up one step in the current pane. I don't have this issue when hitting the left arrow key. I think, that vifm is waiting for another key to be pressed after the `c` key.

What I tried, is to remap all the command starting with `c` by adding the following lines to my vifmrc:
```
noremap ll cl
noremap lo co
noremap lg cg
noremap lp cp
noremap lw cw
noremap Lw Cw
```
But it did not remove the delay.

How can I remove the delay when hitting the `c` key ?

**Comment by xaizek** [comment]
> I think, that vifm is waiting for another key to be pressed after the c key.

Yes, that's what happens.

> But it did not remove the delay.

remap doesn't resolve mappings in right-hand side, but it doesn't remove builtin ones (they can't be removed).

You can set ['timeoutlen'](https://vifm.info/vimdoc.shtml#vifm-%27timeoutlen%27) to say `1` (`0` doesn't seem to work, I tried and it hangs, need to look closer), but then you won't be able to access other ambiguous keys as well. I don't think there is any other solution, and not sure if there is anything to address this in Vim that could be implemented here.

**Comment by nicodebo** [comment]
Thank you for your quick answer.

Decreasing the value of `timeoutlen` isn't a solution for me for the reason you mentioned.
But I would be interested in the `<nowait>` modifier for vifm.

**Comment by xaizek** [comment]
Because I forgot when exactly such delay occurs and `:nmap c` menu lacks separator between user and builtin keys (this is a small bug I just noticed, thanks) I ended up misinforming you, sorry about that.

Your `vifmrc` must include at least one mapping that starts with `c`. Sample vifmrc provides one:
```vim
nnoremap cc cw<c-u>
```
And this is why there is a delay.  It should be gone once you remove or comment out such mappings.  So `<nowait>` isn't necessary for such cases.
